landing/index.html

```diff
@@ -7,6 +7,8 @@
 
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,700;1,400&family=Roboto:wght@400;700&display=swap" rel="stylesheet"> 
     <link rel="stylesheet" href="static/dist/main.css">
+    <link rel="shortcut icon" href="static/img/favicon.ico”" type="image/ico" />
+    <link rel="shortcut icon" href="https://dataocean.us/favicon.ico”" type="image/ico" />
```

Comment:
something strange here
```suggestion
    <link rel="shortcut icon" href="/static/img/favicon.ico" type="image/ico" />
```